### ISSUE TITLE
Documentation of default values is incorrect

### ISSUE THREAD

**Issue by jaimemarijke** [onury/jasmine-console-reporter#10]
**Versions**
Node: 8.10.0
npm: 6.1.0
jasmine: 3.1.0
jasmine-console-reporter: 3.0.2

**Expected**
The documentation describes default values for the options, e.g. that the default `colors` value is 1. 

Thus, I expect that when I try to set up a new reporter without passing in a `color` key it will default to using colors.

**Actual**
When I try to set up a new reporter without passing in a `color` key, it defaults to no colors.

This config has colors in the output:
```
const reporter = new JasmineConsoleReporter({
    colors: 1, // (0|false)|(1|true)|2
    cleanStack: 3, // (0|false)|(1|true)|2|3
    verbosity: 4, // (0|false)|1|2|(3|true)|4
    listStyle: 'indent', // "flat"|"indent"
    activity: true,
    emoji: true, // boolean or emoji-map object
    beep: true,
});
```

This config does not:
```
const reporter = new JasmineConsoleReporter({
    // colors: 1, // (0|false)|(1|true)|2
    cleanStack: 3, // (0|false)|(1|true)|2|3
    verbosity: 4, // (0|false)|1|2|(3|true)|4
    listStyle: 'indent', // "flat"|"indent"
    activity: true,
    emoji: true, // boolean or emoji-map object
    beep: true,
});
```

**Comment by onury** [comment]
Thanks for reporting. I'll check this when I find some time.

**Comment by jaimemarijke** [comment]
Thanks!

**Comment by onury** [comment]
This is now fixed with v3.1.0.
Thanks.
